dockerfiles/dockerfile_ubuntu

```diff
@@ -15,6 +15,11 @@ RUN apt install -y ros-humble-gazebo* && \
     apt install -y ros-humble-urdf && \
     apt install -y ros-humble-urdf-tutorial
 
+RUN apt-get install -y python3 python3-pip && \
+    pip3 install pyserial
+
+ENV PYTHONPATH=/wolvi/src:$PYTHONPATH
+
```

Comment:
This line does not work. Change it to:

_RUN mkdir -p /wolvi/src
ENV PYTHONPATH=/wolvi/src:/usr/local/lib/python3.8/site-packages_